asian-recipes.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
Let's try to make multiple commits next time. One for each file or work scope. E.g. Adding the kung-pao-chicken.html + the kung-pao-chicken.jpeg. That could be one commit.